doc/manual.rst

```diff
@@ -6947,18 +6947,22 @@ language for maximum flexibility:
 - A dot following the hash ``#.`` indicates that the call should use C++'s dot
   or arrow notation.
 - An at symbol ``@`` is replaced by the remaining arguments, separated by commas.
+- An exclamation symbol ``!`` indicates a free function.
```

Comment:
please explain more, it is not easy to understand "free function".

Reply:
added meaning as non-member (and there are examples right below)